velox/functions/sparksql/Register.cpp

```diff
@@ -213,6 +214,9 @@ void registerFunctions(const std::string& prefix) {
   // Register bloom filter function
   registerFunction<BloomFilterMightContainFunction, bool, Varbinary, int64_t>(
       {prefix + "might_contain"});
+
+  // Register decimal functions.
```

Comment:
Since it is not really function in Spark, please use registerExpression as this PR does. https://github.com/facebookincubator/velox/pull/5441/files#diff-19962809ab2250cbf3c99e92b7ba14ae89e2f26ae5e646db3c2b31e67af16e8dR73

Reply:
OK.